code/game/objects/structures/stool_bed_chair_nest/stools.dm

```diff
@@ -153,6 +154,22 @@
 	..(newloc, MATERIAL_SHUTTLE_SKRELL)
 	set_light(1,1,LIGHT_COLOR_CYAN)
 
+/obj/structure/bed/chair/stool/bamboo
+	name = "bamboo stool"
+	desc = "A makeshift bamboo stool with a rustic look."
```

Comment:
```suggestion
	desc = "A makeshift bamboo stool, with a rustic look."
```

Reply:
either is correct here, personally feel like no comma is better for sentence flow